src/index.js

```diff
@@ -1 +1,35 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+
+const tasksList = [
+  { index: 0, completed: false, description: 'Debug weather app code' },
+  { index: 1, completed: false, description: 'Prepare for StandUp meeting' },
+  { index: 2, completed: true, description: 'Study documentation about WebPack' },
+  { index: 3, completed: false, description: 'Organize folders' },
+  { index: 4, completed: false, description: 'Finish ToDo List' },
+];
+
+function createTaskCards() {
+  const taskContainer = document.querySelector('.task-container');
+  const taskTemplate = document.getElementById('task-template');
+  tasksList.forEach((task) => {
+    const taskCard = taskTemplate.content.cloneNode(true).children[0];
+    taskCard.querySelector('.check').checked = task.completed ? 'checked' : '';
+    taskCard.querySelector('.task-description').value = task.description;
+    taskCard.querySelector('.row-task button').setAttribute('data_id', task.index);
+    taskContainer.appendChild(taskCard);
+  });
+}
+
+createTaskCards();
+const tasks = document.querySelectorAll('.task-description');
+tasks.forEach((task) => {
+  task.addEventListener('input', (e) => {
```

Comment:
- [ ] Currently, I can't add a task. The input event seems not to function correctly. When typing in a to-do, nothing is added to the list. Please let's fix it.

Reply:
Hello @FrederickMih. That's correct, that functionality is not implemented because is not part of this requirement, It is part of the next activity:
https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md

Reply:
Hi @VicPeralta. From the project requirements, we are asked to implement a function for adding a new task to the list.